api/spec/v1.yaml

```diff
@@ -3082,6 +3093,37 @@ components:
         verification_level:
           allOf: [$ref: '#/components/schemas/VerificationLevel']
 
+    EvmTokenSwap:
+      type: object
+      required: [pair_address]
+      properties:
+        pair_address:
+          type: string
+        pair_address_eth:
+          type: string
+        factory_address:
+          type: string
+        factory_address_eth:
+          type: string
+        token0_address:
+          type: string
+        token0_address_eth:
+          type: string
+        token1_address:
+          type: string
+        token1_address_eth:
+          type: string
+        create_round:
+          type: integer
+          format: int64
+        reserve0:
```

Comment:
does this correspond to `token0`? If so can we name it something like `token0_reserve`?

Reply:
corresponds to token0. but I prefer the uniswapv2 name reserve0

Reply:
keep these as is